src/API/comments.js

```diff
@@ -0,0 +1,27 @@
+import { client } from '../utils/fetchClient';
+
+export const getComments = async (postId) => {
+  const response = await client.get(`/comments?postId=${postId}`);
+
+  return response.data;
+};
+
+export const addCommentToServer = async (commentData) => {
+  const response = await client.post('/comments', commentData, {
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded'
```

Comment:
The `Content-Type` header is set to `application/x-www-form-urlencoded`. Ensure that this is the required format for your API. If the API expects JSON, you should use `application/json` instead.